front/package.json

```diff
@@ -13,7 +13,7 @@
     "initdb": "env $(cat .env.local) npx tsx admin/db.ts"
   },
   "dependencies": {
-    "@dust-tt/sparkle": "0.1.73",
+    "@dust-tt/sparkle": "0.1.74",
```

Comment:
Approved for the changes but that won't work right? 
You need first to publish the package and then update here on front with change on package.json + on package.lock